### PR TITLE
Complete Overhaul Attempt 2

### PR DESCRIPTION
**Pull request by ThePrivatePanda**
I messed up the first one, sorry

### CODE REVIEW

File: src/bot_base.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+from typing import Union
+from nextcord.abc import GuildChannel
+from nextcord.ext import commands
+from nextcord import Guild, Member, User
+from nextcord.ext import commands
+from aiosqlite.core import Connection
+
+class BotBaseBot(commands.Bot):
```
Comment (Aakash-kun) [inline src/bot_base.py]: Please keep the old class while updating the get/fetch methods. This class also lacks a connection to the Lavalink server.

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+async def startup():
+	print("starting up...")
+	bot.selfrole_view_set = False
+	bot.support_view_set = False
+	bot.owner_id = 707964352199786648
+
+	await bot.wait_until_ready()
+
+	# db stuff
+	bot.db = await aiosqlite.connect(bot.config.get("db_file"))
```
Comment (Aakash-kun) [inline src/main.py:31]: I would like all the bot vars inside the BotBase class as it did it previously. You can set up this connection in the BotBase class with the connection to the Lavalink server in a similar setup func.

File: src/cogs/music_commands.py [hunk 3]
```diff
@@ -9,26 +9,24 @@
 from spotipy.oauth2 import SpotifyClientCredentials
 
 from internal.lavalink_voice_client import LavalinkVoiceClient
-from internal.bot import BotBase
 from internal.player import CustomPlayer
 from internal.load_later_track import LoadLaterTrack
-from config import spotify_client_id, spotify_client_secret
-
+from bot_base import BotBaseBot
```
Comment (Aakash-kun) [inline src/cogs/music_commands.py]: bot_base file is outside the cogs folder, pls confirm the location here.

**Review by Aakash-kun — CHANGES_REQUESTED** [review]
bot_base, config_handler, and database_handler files should be moved to the internal dir.

File: src/bot_base.py (hunk 1 above)
Comment (ThePrivatePanda) [reply src/bot_base.py]: I've updated the reference everywhere I believe, the name better describes the user case so I think it's fine
as for the lack of connection to lavalink server, it's being made in the init of the lavalink_voice_client?

File: src/main.py (hunk 2 above)
Comment (ThePrivatePanda) [reply src/main.py:31]: In ideal case, there should be different bot vraiables for differnet db handlers for each specific table, this should be moved to cog's init later on

File: src/cogs/music_commands.py (hunk 3 above)
Comment (ThePrivatePanda) [reply src/cogs/music_commands.py]: yup it's fine

File: src/main.py (hunk 2 above)
Comment (Aakash-kun) [reply src/main.py:31]: ok, as you see right.

**Review by Aakash-kun — APPROVED** [review]

**Pull request merged**